doc/source/user/absolute_beginners.rst

```diff
@@ -0,0 +1,2451 @@
+  print(arr.size(arr3D))
+
+**Output:**
+
+::
+
+  24
+
+You can also use ``arr.size()`` with 1D arrays:
+
+**Input:**
+
+::
+
+  # Create a 1D array
+  arr = np.array([1, 2, 3, 4, 5, 6, 7, 8])
+
+  # Determine the length
+  print('Length of 1D numpy array : ', arr.size(arr))
+
+**Output:**
+
+::
+
+  Length of 1D numpy array :  8
+
+*Remember that if you check the size of your array and it equals 0, your array is empty.*
+
+Learn more about `finding the size of an array here <https://docs.scipy.org/doc/numpy/reference/generated/numpy.ndarray.size.html>`_ and the `shape of an array here <https://docs.scipy.org/doc/numpy/reference/generated/numpy.ndarray.shape.html>`_.
+
+
+Creating matrices
+-----------------
+
+You can pass Python lists of lists to create a matrix to represent them in NumPy.
+
+::
+
+  np.array([[1,2],[3,4]])
+
+.. image:: images/np_create_matrix.png
+
+Indexing and slicing operations are useful when you're manipulating matrices:
+
+::
+
+  data[0,1]
+  data[1:3]
+  data[0:2,0]
+
+.. image:: images/np_matrix_indexing.png
+
+You can aggregate matrices the same way you aggregated vectors:
+
+::
+
+  data.max()
+  data.min()
+  data.sum()
+
+.. image:: images/np_matrix_aggregation.png
+
+You can aggregate all the values in a matrix and you can aggregate them across columns or rows using the ``axis`` parameter:
+
+::
+  
+  data.max(axis=0)
+  data.max(axis=1)
+
+
+.. image:: images/np_matrix_aggregation_row.png
+
+Once you've created your matrices, you can add and multiply them using arithmetic operators if you have two matrices that are the same size.
+
+::
+
+  data + ones
+
+.. image:: images/np_matrix_arithmetic.png
+
+You can do these arithmetic operations on matrices of different sizes, but only if one matrix has only one column or one row. In this case, NumPy will use its broadcast rules for the operation.
+
+::
+
+  data + ones_row
+
+.. image:: images/np_matrix_broadcasting.png
+
+Be aware that when NumPy prints N-Dimensional arrays, the last axis is looped over the fastest while the first axis is the slowest. That means that:
+
+**Input:** 
+
+::
+
+  np.ones((4,3,2))
+
+Will print out like this:
+
+**Output:**
+
+::
+
+  array([[[1., 1.],
+        [1., 1.],
+        [1., 1.]],
+
+       [[1., 1.],
+        [1., 1.],
+        [1., 1.]],
+
+       [[1., 1.],
+        [1., 1.],
+        [1., 1.]],
+
+       [[1., 1.],
+        [1., 1.],
+        [1., 1.]]])
+
+ 
+There are often instances where we want NumPy to initialize the values of an array. NumPy offers methods like ``ones()``, ``zeros()``,  ``Random Generator``, and even the legacy method ``random.random()`` for these instances. All you need to do is pass in the number of elements you want it to generate.
+
+::
+
+  np.ones(3)
+  mp.zeros(3)
+  np.random.random(3)
+  
+.. image:: images/np_ones_zeros_random.png
+
+You can also use the ``ones()``, ``zeros()``, and ``random()`` methods to create an array if you give them a tuple describing the dimensions of the matrix.
```

Comment:
no `random`?

Reply:
instead:
```
rng = np.random.default_rng()
rng.random()
```

Reply:
I'm not sure if I understood this note, but I changed the existing code block to read this way. Please double-check that I made the correct change!

Reply:
yes it looks correct, resolving

Reply:
Change looks good. Unfortunately the image still contains `np.random.random`. Leaving this comment open for now, we're going to have to deal with those images at some point.